docs/fastdds/getting_started/simple_app/includes/task_run.rst

```diff
@@ -0,0 +1,16 @@
+Putting all together
+""""""""""""""""""""
+
+Finally, from the build directory, run the publisher and subscriber applications from various terminals.
+
+.. code-block:: bash
+
+    ./DDSHelloWorldPublisher
+    ./DDSHelloWorldSubscriber
+
+.. raw:: html
+
+    <video width="1000" height="563" autoplay loop>
+        <source src="../../../_static/simple_pubsub.mp4">
+        Your browser does not support the video tag.
+    </video>
```

Comment:
For some reason is not animated for me. Furthermore, why in _static and not in figures=